### DISCUSSION TITLE
Add PV Power sensor to sofar_g3hyd.yaml

### DISCUSSION THREAD

**Discussion by jandeuss** [davidrapan/ha-solarman#228] [Ideas]
Can you please add a sensor to the file in the inverter DC section.

- name: "PV Total Power"
        class: "power"
        state class: "measurement"
        uom: "W"
        scale: 100
        rule: 1
        registers: [0x05C4]
        icon: "mdi:solar-power"

This adds the sensor of Total PV Power of a master / slave System.

thanks a lot.

**Comment by davidrapan** [comment]
Sure thing: [feat: Add PV Power to sofar_g3hyd profile](https://github.com/davidrapan/ha-solarman/commit/49159e397f34b0c904c2721cf57c7e7fcb987fc5)
